src/main/java/com/cognifide/aemrules/checks/AdministrativeAccessUsageCheck.java

```diff
@@ -57,31 +60,40 @@
         .put("getAdministrativeResourceResolver", "getServiceResourceResolver")
         .build();
 
+    private static final List<MethodMatcher> matchers = ImmutableList.of(
+        MethodMatcher.create(
+            MethodNamePredicate.is("loginAdministrative"),
+            TypePredicate.is("org.apache.sling.jcr.api.SlingRepository"),
```

Comment:
Well, I think that for readability purpose, it would be great if we can differentiate `OwnerType` and `ParameterType` as it's not really visible her what's happening.